app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,45 @@
+<nav class="navbar navbar-expand-lg bg-body-tertiary mb-4">
+   <div class="container">
+   <%= link_to "Photogram", root_path, class: "navbar-brand" %>
+    <!-- <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button> -->
+   <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <ul class="dropdown-menu">
+            <li><a class="dropdown-item" href="/photos">Photos</a></li>
+            <li><a class="dropdown-item" href="/comments">Comments</a></li>
+            <li><a class="dropdown-item" href="/follow_requests">Follow requests</a></li>
+            <li><a class="dropdown-item" href="/likes">Likes</a></li>
+
+          </ul>
+        </li>
+      </ul>
+
+      <ul class="navbar-nav">
+        <% if user_signed_in?%>
+
+              <li class="nav-item">
+                <%= link_to "Edit profile", edit_user_registration_path(current_user), class: "nav-link" %>
```

Comment:
Your edit user route is not currently passing the rake grade task. When I attempt to navigate to the edit user profile page, the route redirects to "users/edit.2". However, the route should redirect to "users/edit". Your route is also adding the user's id #. Please review this line and lmk if you have any questions on how to debug!